modules/virt-creating-a-primary-udn.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-creating-a-primary-udn_{context}"]                                  
+= Creating a primary namespace-scoped user-defined network
+
+You can create an isolated primary network in your project namespace. You must use the OVN-Kubernetes layer 2 topology and enable persistent IP address allocation in the user-defined network (UDN) configuration to ensure VM live migration support.
+
+.Prerequisites
+. You have installed the OpenShift CLI (`oc`).
+. You have created a namespace and applied the `k8s.ovn.org/primary-user-defined-network` label.
```

Comment:
@maiqueb This is new to me.
Until now, we only needed to create a UDN in a certain namespace, in order to make this namespace a UDN one.
Do you know anything else?

Reply:
Yes, this is new.

You need to do this upon namespace creation, since the namespace is not mutable. 

Hence, the user, when they create the namespace, must already label the namespace appropriately. 

Afterwards, they'll create the UDN. This was the way we found of preventing the UDN / workload race where the workload would be processed before the UDN, and the workload would be granted a default cluster network attachment instead.

Reply:
Is this also true for C-UDN?